common/helpers.js

```diff
@@ -6,7 +6,7 @@ const moment = require('moment');
 const dataGuidsHostPrefix = 'dg.';
 const dosDataObjectPathPrefix = '/ga4gh/dos/v1/dataobjects/';
 const drsDataObjectPathPrefix = '/ga4gh/drs/v1/objects/';
-const jadeDataRepoHostRegex = /jade.*\.datarepo-.*\.broadinstitute\.org/;
+const jadeDataRepoHostRegex = /.*data.*\.(broadinstitute\.org|envs-terra\.bio)$/;
```

Comment:
Bold. I like it I think. I don't think we'll hit the pathological combo of:
1) a new Terra host with `data` somewhere in the name
2) that new Terra host being used in a `dos://` or `drs://` path and passed to Martha